.babelrc.js

```diff
@@ -2,9 +2,9 @@ let ignore = [`**/dist`]
 
 // Jest needs to compile this code, but generally we don't want this copied
 // to output folders
-if (process.env.NODE_ENV !== `test`) {
-  ignore.push(`**/__tests__`)
-}
+// if (process.env.NODE_ENV !== `test`) {
+//   ignore.push(`**/__tests__`)
+// }
```

Comment:
This breaks linting as babel will return an empty ast. Packages should be build with `babel src --out-dir . --ignore \"**/__tests__\"`